.github/workflows/CI.yaml

```diff
@@ -35,10 +35,16 @@ jobs:
       - name: Clean neo4j from the cache
         if: success() || failure() # run this step even if previous step failed
         run: (find ~/.gradle/caches -name "*neo4j*" -exec rm -rf {} \;) || echo "All neo4j files cleaned"
+      - name: Check procedures included in apoc full
+        run: git diff --exit-code full/src/main/resources/extended.txt || (echo "extended.txt file does not contain all changes" && exit -1)
```

Comment:
The build re-generates the `extended.txt` file. If it contains changes with respect to the one the repo contains, the build will tell us it is incomplete and the changes we have to make (via `git diff`)